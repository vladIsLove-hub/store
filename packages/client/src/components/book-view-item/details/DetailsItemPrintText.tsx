import React from 'react';

import { DetailsItemPrintTextProps } from './details.types';

const DetailsItemPrintText: React.FC<DetailsItemPrintTextProps> = ({ text }) => {
  return (
    <div className="book-view__details-list-item-right">
      <span data-testid="item-value" title={text} className="book-view__details-list-item-value">
        {text}
      </span>
    </div>
  );
};

export default DetailsItemPrintText;
