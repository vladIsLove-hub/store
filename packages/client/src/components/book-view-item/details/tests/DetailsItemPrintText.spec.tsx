import React from 'react';
import { render } from '@testing-library/react';

import DetailsItemPrintText from '../DetailsItemPrintText';

describe('DetailsItemPrintText', () => {
  it('should render cuurently', () => {
    const { getByText } = render(<DetailsItemPrintText text="test" />);
    expect(getByText('test')).toBeInTheDocument();
  });
  it('DetailsItemPrintText snapshot', () => {
    const { getByText } = render(<DetailsItemPrintText text="test" />);
    expect(getByText('test')).toMatchSnapshot();
  });
});
