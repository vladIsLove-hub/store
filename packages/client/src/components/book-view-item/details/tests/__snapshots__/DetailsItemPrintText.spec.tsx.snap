// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsItemPrintText DetailsItemPrintText snapshot 1`] = `
<span
  class="book-view__details-list-item-value"
  data-testid="item-value"
  title="test"
>
  test
</span>
`;
